GameData/kappa-ray/kapparay_BioMass_waterTanks.cfg

```diff
@@ -0,0 +1,37 @@
+// Add KappaRay handler to BioMass water tanks
+
+@PART[H2ORadialLong]:FOR[kapparay]
```

Comment:
Actually I thought I'd rather make tanks of things shield proportional to their fill level.  So instead of attaching a ModuleKappaRayShielding to each part, you'd add a shielding coefficient /k/ to each *resource definition*, and then the part will shield proportional to, say, ∛(km) where /m/ is the mass of that resource present in the tank.  Cube roots seem appropriate because absorption depends on the linear distance through the material, and mass scales as volume.  Mass is a good choice because denser materials are usually inherently better shielding, so a wide range of resources will have /k/ values of similar scale.  Also, because absorption is a logarithmic process, the final absorpCoeff would be something like e^∛(km).

But I do approve of the idea!  Water should definitely act as shielding, and is used that way in most real-world design proposals for interplanetary vessels.  (So is food, which is made of mostly water, as is the, let's say, result of food.)